### DISCUSSION TITLE
Restoring an old session/key/identity?

### DISCUSSION THREAD

**Discussion by ProxyCell** [MihanEntalpo/cryptboard.io#79] [General]
Is it possible for me to restore an old session/key/identity on cryptboard? Or do I lose everything after I close out my browser? I don't really see an option to save the private key anywhere.

**Comment by MihanEntalpo** [comment]
Hello. After you closing browser nothing happens.
If you reopen it - everything would work as intended.
But if you would hold it closed for some time, your session would expire, and you cannot use your user_id again. Keys are stored in your browser's local storage, so that you can get private and public rsa keys from local storage.
Anyway, cryptboard.io doesn't aimed to be persistent chat/clipboard, but is a temporary, almost anonymous service.

**Comment by MihanEntalpo** [comment]
Please, describe your use-case, so I can give you solution, or think about how much would it take to make changes that you need
